### PR TITLE
rpm: add --changelog example

### PR DESCRIPTION
**Pull request by mfrw**
Signed-off-by: Muhammad Falak R Wani <falakreyaz@gmail.com>

- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/rpm.md [hunk 1]
```diff
@@ -30,3 +30,7 @@
 - Show changed, missing and/or incorrectly installed files of matching packages:
 
 `rpm --verify --all '{{php-*}}'`
+
+- Display changelog of a specific package:
```
Comment (marchersimon) [inline pages/linux/rpm.md:34]:
```suggestion
- Display the changelog of a specific package:
```

File: pages/linux/rpm.md [hunk 2]
```diff
@@ -30,3 +30,7 @@
...
+
+`rpm --query --changelog {{package_name}}`
```
Comment (marchersimon) [inline pages/linux/rpm.md:36]: The other examples use `package_name.rpm`. Which one should we use?

Comment (mfrw) [reply pages/linux/rpm.md:36]: In this case `package_name` is the name of the package without `.rpm`.
It is the installed `rpm` that is on the system.
It is one of the packages of the output from:
`rpm --query --all`

**Review by marchersimon — APPROVED** [review]
Thanks @mfrw

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**